doc/user/content/sql/patterns/percentiles.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "Tracking percentiles"
+description: "How to efficiently track percentiles in Materialize."
+menu:
+  main:
+    parent: 'sql-patterns'
+---
+
+Percentiles are often used to understand and interpret data. Informally speaking, the k-th percentile is the value such that k percent of all values are lower than the percentile.
+
+A naive way to compute percentiles is to simply order all values and pick the value at the position of the corresponding percentile. This can be easily expressed in Materialize through `order by`, `offset`, and `limit`. But this way of computing percentiles keeps all values around and therefore requires memory that linearly grows with the number of values that are tracked. There are better suited alternatives that reduce the required memory footprint, two of which are explained below: histograms and High Dynamic Range (HDR) histograms.
```

Comment:
I've cleaned up the intro a bit. But I did not remove it entirely because I feel like it adds additional context that makes it easier for people to choose between histograms (more memory but precise) and hdr histograms (potentially less memory but an approximation).